package.json

```diff
@@ -16,15 +16,16 @@
         "url": "https://github.com/hoho/gulp-dedupe/issues"
     },
     "dependencies": {
-        "gulp-util": "~3.0.1",
-        "through": "~2.3.6",
-        "lodash.defaults": "~2.4.1",
         "colors": "~1.0.2",
-        "diff": "~1.0.8"
+        "diff": "~1.0.8",
+        "lodash.defaults": "~2.4.1",
+        "plugin-error": "^1.0.1",
+        "through": "~2.3.6"
     },
     "devDependencies": {
-        "mocha": "*",
-        "should": "*"
+        "mocha": "3.5.3",
```

Comment:
Let's maybe have "~3.5.3"?

Reply:
Why? :)
Is it not better to just pin an exact version in this case: the latest version that is still compatible with node 0.10, but has as many features and bug fixes as possible. So basically exactly version 3.5.3.
~3.5.3 will install 3.5.3 anyway.

Reply:
Not in this case, because Mocha dropped support for Node 0.10. But I changed it anyway.

Reply:
Why in the world anybody needs 0.10 support in 2018?

Reply:
By all means drop it. I just looked at [this](https://github.com/hoho/gulp-dedupe/blob/efd9e149ebe73ed8dd867e9027c07566ee5f0844/package.json#L33) and [this](https://github.com/hoho/gulp-dedupe/blob/efd9e149ebe73ed8dd867e9027c07566ee5f0844/.travis.yml#L4) and concluded this package supports 0.10.

In that case, indeed, no reason to pin Mocha to a specific version. But rather update the `engines` field and travis config. Which node versions DO you want to support?